lightning/src/ln/onchaintx.rs

```diff
@@ -709,4 +778,28 @@ impl<ChanSigner: ChannelKeys> OnchainTxHandler<ChanSigner> {
 			self.pending_claim_requests.remove(&req);
 		}
 	}
+
+	pub(super) fn provide_latest_local_tx(&mut self, tx: LocalCommitmentTransaction) {
+		self.prev_local_commitment = self.local_commitment.take();
+		self.local_commitment = Some(tx);
+	}
+
+	pub(super) fn get_fully_signed_local_tx(&mut self, channel_value_satoshis: u64) -> Option<Transaction> {
+		if let Some(ref mut local_commitment) = self.local_commitment {
```

Comment:
Note that you can/should be able to unwrap here - ChannelMonitors are created with initial commitment transactions in them now (its still a separate call, but we should fix that).

Reply:
Hmmm letting this one, in fact we have empty-local-commitment-transaction ChannelMonitor in get_outbound_funding_created, hit by one of our test